packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -526,15 +526,20 @@ export class ScmResourceComponent extends ScmElement<ScmResourceComponent.Props>
         const icon = labelProvider.getIcon(resourceUri);
         const color = decoration && decoration.colorId ? `var(${colors.toCssVariableName(decoration.colorId)})` : '';
         const letter = decoration && decoration.letter || '';
-        const tooltip = decoration && decoration.tooltip || '';
+        const tooltip: string = decoration && decoration.tooltip || '';
```

Comment:
Nitpick: As far as I can tell, this `string` typing is not necessary. TypeScript should be able to infer the type correctly as `string`. Tested with TS 4.3.2:

![image](https://user-images.githubusercontent.com/4377073/125953212-2dfe8e1f-9bd0-4dfa-bd17-2361f992abad.png)

Reply:
For some reason if was not inferring it for me (`any`), but I can confirm it works now when restarting the language server :+1: